engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,28 +238,59 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float ratio) {
+        return (float) Math.pow(ratio, 0.74f) * 6f + 4f;
+    }
```

Comment:
The default jump speed for the player is set in the prefab to 12, thus calculating with 10 here sets the default values off (sorry for not catching this earlier). I like the fact the we have a constant value here to ensure that the player can always jump some minimum height. 
When deriving this from the default jump speed I would also increase the influence of the constant and reduce the influence of the first part:
```suggestion
    private float getJumpSpeed(float ratio, float defaultValue) {
        return (float) Math.pow(ratio, 0.74f) * 0.4f * defaultValue + 0.6f * defaultValue;
    }
```